File: app/src/main/java/com/kelsos/mbrc/data/ProtocolPayload.java
package com.kelsos.mbrc.data;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

@JsonInclude(JsonInclude.Include.NON_NULL)

@JsonPropertyOrder({
    "no_broadcast",
    "protocol_version"
})
public class ProtocolPayload {

  @JsonProperty("no_broadcast")
  private Boolean noBroadcast;
  @JsonProperty("protocol_version")
  private Float protocolVersion;

  @JsonProperty("no_broadcast") public Boolean getNoBroadcast() {
    return noBroadcast;
  }

  @JsonProperty("no_broadcast") public void setNoBroadcast(Boolean noBroadcast) {
    this.noBroadcast = noBroadcast;
  }

  @JsonProperty("protocol_version") public Float getProtocolVersion() {
    return protocolVersion;
  }

  @JsonProperty("protocol_version") public void setProtocolVersion(Float protocolVersion) {
    this.protocolVersion = protocolVersion;
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/data/ProtocolPayload.kt
package com.kelsos.mbrc.data

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonPropertyOrder

@JsonInclude(JsonInclude.Include.NON_NULL)

@JsonPropertyOrder("no_broadcast", "protocol_version")
class ProtocolPayload {

  @JsonProperty("no_broadcast")
  var noBroadcast: Boolean? = null
  @JsonProperty("protocol_version")
  var protocolVersion: Float? = null
}
